src/Query/Operation/Get.php

```diff
@@ -4,11 +4,12 @@
 namespace TBolier\RethinkQL\Query\Operation;
 
 use TBolier\RethinkQL\Message\MessageInterface;
+use TBolier\RethinkQL\Query\AbstractQuery;
 use TBolier\RethinkQL\Query\QueryInterface;
 use TBolier\RethinkQL\RethinkInterface;
 use TBolier\RethinkQL\Types\Term\TermType;
 
-class Get extends AbstractOperation
+class Get extends AbstractQuery
```

Comment:
Get needs to support other operations like `Update` and `Update`, etc. so it needs to extend AbstractQuery

Reply:
Fixed